docs/concepts/trading-on-vega/discounts-rewards.md

```diff
@@ -27,6 +27,30 @@ You can see what the current program offers by checking the [Referrals page on C
 * [Spec: Technical design of the referral program ↗](https://github.com/vegaprotocol/specs/blob/master/protocol/0083-RFPR-on_chain_referral_program.md).
 :::
 
+### Referral sets
+To benefit from referral program perks, you'll need to either create a referral set, or join one. Referral sets can also be made into [teams](#teams-and-games), which get access to rewards targeted towards team members.
+
+A referral set is made up of the participant who created the set, known as the referrer, and all the referees who signed up using the referral code. Each referral set only has one referrer, but the number of referees is unlimited.
+
+:::tip Try it out
+Create a referral code, or enter a referral code you've been given on the [referrals section of Console ↗](https://vegafairground.eth.limo/#/referrals).
```

Comment:
**nitpick (non-blocking)**: note for us in future (major/minor version) network upgrades: we'll need to update these links. The build scripts won't currently do it for us.